frontend/packages/dev-console/src/components/pipelines/resource-overview/DynamicResourceLinkList.tsx

```diff
@@ -25,9 +23,9 @@ const DynamicResourceLinkList: React.FC<DynamicResourceLinkListProps> = ({
     return null;
   }
   return (
-    <div className="odc-dynamic-resource-link-list">
```

Comment:
You caused this issue on the Trigger Templates details page... careful removing styles without tracing all usages.

![Screen Shot 2020-09-03 at 1 14 07 PM](https://user-images.githubusercontent.com/8126518/92146257-7090fd80-ede7-11ea-8b0c-2e9a91c7df1d.png)

Maybe make it a prop to disable spacing?

Reply:
fixed it ! using a prop to add spacing